app/controllers/meals_controller.rb

```diff
@@ -6,12 +6,13 @@ class MealsController < ApplicationController
   # GET /meals.json
   def index
     @meals = current_user.meals.includes(:recipe)
-    @meals_json = @meals
-      .collect{ |meal| {
-      title: meal.recipe.name,
-      start: meal.date.strftime("%Y-%m-%d"),
-      url: edit_meal_url(meal),
-      className: meal.time
+    @meals_json = @meals.
+      sort_by(&:time_position).
+      collect{ |meal| {
```

Comment:
Avoid using {...} for multi-line blocks.<br>Prefer `map` over `collect`.<br>Block body expression is on the same line as the block start.<br>Align the operands of an expression in an assignment spanning multiple lines.